jfrog-applications/ide/visual-studio-code/extension-settings.md

```diff
@@ -21,6 +21,23 @@ If your JFrog environment is behind an HTTP/S proxy, follow these steps to confi
 
 * Alternatively, you can use the HTTP\_PROXY and HTTPS\_PROXY environment variables.
 
+#### Downloading External Resources Through Artifactory
+
+JFrog VS Code extension required necessary resources for scanning vulnerabilities in your project.
+By default, the JFrog extension downloads the necessary resources needed from https://releases.jfrog.io. If the machine that runs JFrog extension has no access to it, follow these steps to allow the tools to be downloaded from an Artifactory instance, which the machine has access to:
```

Comment:
...If the machine that runs JFrog extension...
-->
...If the machine that runs VS Code...

...to allow the tools to be downloaded from an Artifactory...
-->
...to allow the tools to be downloaded through an Artifactory...